packages/playwright/src/util.ts

```diff
@@ -35,6 +35,12 @@ export function filterStackTrace(e: Error): { message: string, stack: string } {
     return { message: e.name + ': ' + e.message, stack: e.stack || '' };
 
   const stackLines = stringifyStackFrames(filteredStackTrace(e.stack?.split('\n') || []));
+  let cause: Error = e.cause as any;
+  while (cause) {
+    stackLines.push('[cause]: ' + cause.toString());
```

Comment:
I am somewhat worried that various tools expect stack trace lines to start with `     at`, and we will break them.